src/main/scala/edu/ie3/util/scala/quantities/QuantityUtil.scala

```diff
@@ -291,4 +292,18 @@ object QuantityUtil {
       if (q.isGreaterThan(other)) q else other
     }
   }
+
+  implicit class RichUnit[Q <: Quantity[Q]](
+      private val unit: measure.Unit[Q]
+  ) extends AnyVal {
+
+    /** Transform some power unit to given unit with the same prefix
+      * @param targetUnit
+      *   the target system unit
+      * @return
+      *   this unit converted to given
+      */
+    def toEquivalentIn(targetUnit: measure.Unit[Q]): measure.Unit[Q] =
+      targetUnit.transform(unit.getConverterTo(unit.getSystemUnit))
+  }
```

Comment:
I think it makes sense to transfer this class (as well as the RichQuantity) into the PowerSystemUtils as it might come in handy for other projects as well and we already have som Quantity related utils there.

Reply:
https://github.com/ie3-institute/PowerSystemUtils/pull/289

If you want to make a quick review we can add it while were at it

Reply:
Ok, that was really quick. I removed `RichUnit` and `RichQuantity` here and replaced all calls with ones to the equivalents in PSU